dist/stat_data.py

```diff
@@ -998,6 +1000,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_hs_write_squash', 'history store table writes requiring squashed modifies'),
     CacheStat('cache_inmem_split', 'in-memory page splits'),
     CacheStat('cache_inmem_splittable', 'in-memory page passed criteria to be split'),
+    CacheStat('cache_multi_block_reconciliation', 'multiple block reconciliation whilst checkpoint is running'),
```

Comment:
I would name it cache_eviction_blocked_multi_block_reconcilation_during_checkpoint

Reply:
fixed https://github.com/wiredtiger/wiredtiger/pull/10046/commits/a8303dcd3f551be75085fcc46e7412e18e18c8fe